latest/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Redirecting</title>
+  <script>
+    var currentVersion = window.location.pathname.split("/")[1];
```

Comment:
The value is static, always "latest". So:
```suggestion
    const currentVersion = "latest";
```